src/sentry/integrations/github/integration.py

```diff
@@ -48,14 +49,17 @@
 }
 
 
-class GitHubIntegration(Integration, RepositoryMixin):
+class GitHubIntegration(Integration, GitHubIssueSync, RepositoryMixin):
 
     def get_client(self):
         return GitHubAppsClient(external_id=self.model.external_id)
 
     def get_repositories(self):
         return self.get_client().get_repositories()
 
+    def format_external_key(self, data):
```

Comment:
I might name this method something like `make_external_key`, since formatting to me implies something more visual, and especially because of the nature of this, it would be easy to think that's all this is for.

Reply:
Can this be part of the `GitHubIssueSync` class?